.fleet/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "toolchains": []
```

Comment:
Can we remove this?

Reply:
@Soare-Robert-Daniel

Reply:
Yes. And better put it to `.gitignore` since it is IDE-related.

Reply:
Can you make the changes? Delete it and put it in gitignore?